doc/lua_api.txt

```diff
@@ -5098,6 +5098,11 @@ Definition tables
         y_min = 1,
         y_max = 31000,
     --  ^ Lower and upper limits for biome.
+        x_min = -31000,
+        x_max = 31000,
+        z_min = -31000,
+        z_max = 31000,
+    --  ^ Horizontal limits for biome. min/max default to -/+31000.
```

Comment:
Having these as two vectors, `minp` and `maxp` would be more readable as the coordinates can be compared easily vertically between the two code lines.

Reply:
Yes i considered that, i chose this way for backwards compatibility and because for the majority of the time only y_min and y_max will be used, vectors would always involve X and Z whereas this way those can be forgotten and left to defaults.

Reply:
In that case, a new `getv3field_default` would be handy. Values like `minp = { y = 0, z = -1000 }, maxp = { x = 0, y = 100 }` could be used in that case. Not set coordinate values would have a default value of `31000`, `-31000` respectively.

Reply:
Hmm not sure, 95-99% of the time only 'y_min' and 'y_max' will be used in biome registrations.
Would your suggestion accept simple 'y_min' and 'y_max'?

Reply:
Yes, if there's a line of compatibility code: (untested)
```C++
b->minp = getv3s16field_default(L, index, "minp", v3s16(-31000));
getintfield(L, index, "y_min", b->minp.Y); // Overwrite if existing
b->maxp = getv3s16field_default(L, index, "maxp", v3s16(-31000));
getintfield(L, index, "y_max", b->maxp.Y); // Overwrite if existing
```

Reply:
Ok something to consider, PR is now WIP.